### DISCUSSION TITLE
Create beta release of AnyVar

### DISCUSSION THREAD

**Discussion by ehclark** [biocommons/anyvar#111] [General]
Can we create a tag/release of AnyVar.  Maybe a 0.9.0.  And have it pinned to the 2.0.0-a12 version of vrs-python?  It would be helpful for me as we release internal services that use AnyVar to be able to pin those services to a specific version.

**Comment by jsstevenson** [comment]
No objections here. Pinning specific VRS-Python 2a versions makes sense for a release. @ehclark are you wanting to push to PyPI or is a GitHub release sufficient? I think you have permissions for the latter already but we can ask Alex to fix if not.

* Noticed that our CI/CD is a little bare, so I grabbed parts of the biocommons template in #112 . We do have a fair amount of leeway to deviate from the rest of the biocommons repos, though, if it makes sense. I think there's a small amount of configuration to be done on the PyPI side to make the release workflow push correctly.

**Reply by ehclark** [reply]
Just a GitHub release is sufficient for me.  Basically I want a stable tag I can point my Docker image builds at.  I'm not well versed in how this process normally works.  Since I want to fix the vrs-python version dependency, would it make sense for me to create a v0.9.0 release branch, commit the change to the pyproject.toml file and then tag the branch?

**Reply by jsstevenson** [reply]
In biocommons projects (which, again, we are definitely free to deviate from if there's a good reason) we typically just make tags against commits on `main` (well, really, we make GitHub releases because this autogenerates a changelog and can be helpful for sphinx autodocs, but a GitHub release adds a tag as a side effect). Creating a release then kicks off the GitHub action to upload to PyPI.

I've just made a PR (#114 ) to pin the VRS-Python version and I can just put a release up once you approve, if that sounds good.

**Reply by ehclark** [reply]
Sounds great to me.  Thanks @jsstevenson

**Reply by jsstevenson** [reply]
👍 pushed a release: https://github.com/biocommons/anyvar/releases/tag/0.9.0
